src/Modal.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
         var closeDialog = function() {
             ReactDOM.unmountComponentAtNode(self.getOrCreateContainer());
 
-            if (props && props.onFinished) props.onFinished.apply(null, arguments);
+            if (props && props.onFinished) props.onFinished.call(null, false);
```

Comment:
Exactly, I did some searching but couldn't find a case where that was actually used (the event that caused the dialog to be closed)